src/sentry/static/sentry/app/actionCreators/repositories.tsx

```diff
@@ -12,17 +12,17 @@ export function getRepositories(api: Client, params: ParamsGet) {
   const {orgSlug} = params;
   const path = `/organizations/${orgSlug}/repos/`;
 
-  RepoActions.loadRepos(orgSlug);
+  RepositoryActions.loadRepositories(orgSlug);
```

Comment:
```suggestion
  RepositoryStore.state.repositoriesLoading = true;
  RepositoryStore.state.orgSlug = orgSlug;
  RepositoryActions.loadRepositories(orgSlug);
```

Is the change I was trying to suggest earlier. I think having this here before the action would better encapsulate the intent of the action creator and make it simpler to re-use now and in the future.

Reply:
Ah I understand now. Excellent suggestion.